### DISCUSSION TITLE
Vue starter package

### DISCUSSION THREAD

**Discussion by garyross4** [mdbootstrap/TW-Elements#1601] [Share ideas & request features]
I've seen your Vue integration guide [here](https://tailwind-elements.com/docs/standard/integrations/vue-integration/).

Everything works fine & I actually like that you explain step by step, what, how, and why you do.

Nevertheless I think it would be useful to have a pre-integrated starter package somewhere.

**Comment by filipkappa** [comment]
As of now, the package is not ready. You can sign up for a waiting list to get notified when we release a dedicated package for Vue:
https://tailwind-elements.com/vue/
